module_info_import/models/module_information.py

```diff
@@ -1,43 +1,40 @@
-import requests
-from odoo import models, api, release, _
-from odoo.exceptions import UserError
 import logging
 
+import requests
+import yaml
+
+from odoo import _, api, models
 
 _logger = logging.getLogger(__name__)
 
 
 ERROR_MESSAGE = _("There is an issue with module information synchronization")
+VERSIONS = ["14.0", "15.0", "16.0"]
```

Comment:
VERSIONS = ["14.0", "15.0", "16.0"]
SELECTION_VERSION = [(v, v) for v in VERSIONS]